### PR TITLE
Bug: 1695205 Reset progressing transition time after upgrades.

### PR DESCRIPTION
**Pull request by dgoodwin**
CCO does not have any components to upgrade, so the first time we go to
update status and see that our running version has changed, update the
progressing last transition time to indicate something happened.

### CODE REVIEW

File: pkg/controller/credentialsrequest/status_test.go [hunk 1]
```diff
@@ -182,20 +205,58 @@ func TestClusterOperatorVersion(t *testing.T) {
...
+			assert.True(t, foundVersion, "didn't find an entry named 'operator' in the version list")
+
+			progCond := findClusterOperatorCondition(clusterop.Status.Conditions,
+				configv1.OperatorProgressing)
+			assert.NotNil(t, progCond)
+			if test.expectProgressingTransition {
+				assert.True(t, progCond.LastTransitionTime.Time.After(
+					test.currentProgressingLastTransition.Time))
+			} else {
+				assert.Equal(t, progCond.LastTransitionTime.Time.Format(time.UnixDate),
```
Comment (joelddiaz) [inline pkg/controller/credentialsrequest/status_test.go]: assert.Equal() wants the expected value to be the second param, and the tested value to be the third param.

File: pkg/controller/credentialsrequest/status_test.go [hunk 2]
```diff
@@ -182,20 +205,58 @@ func TestClusterOperatorVersion(t *testing.T) {
...
 			if err := fakeClient.Get(context.TODO(), client.ObjectKey{Name: cloudCredClusterOperator}, clusterop); err != nil {
 				t.Errorf("error fetching clusteroperator object: %v", err)
-			} else {
-				foundVersion := false
-				for _, version := range clusterop.Status.Versions {
-					if version.Name == "operator" {
-						foundVersion = true
-						assert.Equal(t, test.releaseVersionEnv, version.Version)
-					}
+				return
```
Comment (joelddiaz) [inline pkg/controller/credentialsrequest/status_test.go]: nit. you can probably just assert.NoError() on that fakeClient.Get() call above

**Comment by csrwng** [comment]
sorry @joelddiaz :)

**Comment by csrwng** [comment]
cancel hold when you're happy with it

**Comment by dgoodwin** [comment]
updated, thanks!

**Pull request merged**